### PR TITLE
docs: add ppas and fix typos

### PR DESCRIPTION
**Pull request by orndorffgrant**
## Proposed Commit Message

> docs: small readme fixes
>
> * Fixes some small grammatical mistakes.
> * Adds details of the three PPAs available for the client.
> * Adds the `make testdeps` step to the Testing section with a brief explanation.

## Test Steps

Read the proposed changes to the readme and decide if they make sense or not.

## Desired commit type::

 - [x] Squash and merge: Using the "Proposed Commit Message"
 - [ ] Create a merge commit and use "Proposed Commit Message"
 - [ ] Rebase and merge, no merge commit, rely only on existing commit messages

## Checklist:

 - [ ] I have updated or added any unit tests accordingly
 - [ ] I have updated or added any integration tests accordingly
 - [x] I have updated or added any documentation accordingly

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,6 +20,15 @@ The client comes pre-installed on all Ubuntu systems in the debian packages
...
 
+Additionally, there are 3 PPAs with different release channels of the Ubuntu Advantage Client:
+
+1. Stable: This contains stable builds only.
+    - add with `sudo add-apt-repository ppa:ua-client/stable`
+2. Staging: This contains builds that are being prepared for release to stable.
+    - add with `sudo add-apt-repository ppa:ua-client/staging`
+3. Daily: This PPA is updated every day with the latest changes.
+    - add with `sudo add-apt-repository ppa:ua-client/daily`
+
```
Comment (orndorffgrant) [inline README.md:31]: First of all: is this the gist of what we want to add for these and is it in an appropriate place?

I made a best guess for these descriptions based on the names. Please let me know if anything is wrong.

File: README.md [hunk 2]
```diff
@@ -120,7 +129,15 @@ The following describes the intent of UA client related directories:
 
 ## Testing
 
-All unit and lint tests are run using tox:
+All unit and lint tests are run using `tox`. We also use `tox-pip-version` to specify an older pip version as a workaround: we have some required dependencies that can't meet the strict compatibility checks of current pip versions.
```
Comment (orndorffgrant) [inline README.md:132]: This is my attempt at a short description of why we have `tox-pip-version` based on my memory of what @lucasmoura was saying yesterday. Please chime in if its incorrect/too long/unnecessary/etc.

File: README.md [hunk 3]
```diff
@@ -181,9 +198,9 @@ Furthermore, when developing/debugging a new scenario:
 
  1. Add a `@wip` tag decorator on the scenario
  2. To only run @wip scenarios run: `tox -e behave-20.04 -- -w`
- 4. If you want to use a debugger:
-    a. Add ipdb to integration-requirements.txt
-    b. Add ipdb.set_trace() in the code block you wish to debug
+ 3. If you want to use a debugger:
+    1. Add ipdb to integration-requirements.txt
+    2. Add ipdb.set_trace() in the code block you wish to debug
```
Comment (orndorffgrant) [inline README.md:203]: The letters here weren't rendering properly. I prefer the look of the letters in plaintext, but when rendered as html, the sub-ordered list looks better to me. LMK if you'd rather I change it back

File: README.md (hunk 2 above)
Comment (lucasmoura) [reply README.md:132]: I think this is a great description of the problem. Thanks @orndorffgrant

**Review by lucasmoura — APPROVED** [review]

**Review by lucasmoura — CHANGES_REQUESTED** [review]
@orndorffgrant I think we can also add the `make testdeps` step before running the uaclient tests.
After adding that, I can merge this PR

**Comment by orndorffgrant** [comment]
Thanks @lucasmoura! the `make testdeps` step is there underneath the comment about tox-pip-version: https://github.com/canonical/ubuntu-advantage-client/pull/1496/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R137

My comment above probably distracted from it - is it in the right place?

**Review by lucasmoura — APPROVED** [review]
You are totally right here @orndorffgrant. My mistake here

**Comment by lucasmoura** [comment]
This PR is touching only README files, so I will not wait for the CI run on it

**Pull request merged**